clinica/pipelines/pet_volume/pet_volume_utils.py

```diff
@@ -10,6 +10,29 @@
 __status__ = "Development"
 
 
+def get_pipeline_parameters(group_id,
+                            psf_tsv=None,
+                            pet_tracer=None,
+                            mask_tissues=None,
+                            mask_threshold=None,
+                            pvc_mask_tissues=None,
+                            smooth=None,
+                            atlases=None):
+
+    list_atlases = ['AAL2', 'LPBA40', 'Neuromorphometrics', 'AICHA', 'Hammers']
+    parameters = {
+        'group_id': group_id,
+        'psf_tsv': psf_tsv or None,
+        'pet_tracer': pet_tracer or 'fdg',
+        'mask_tissues': mask_tissues or [1, 2, 3],
+        'mask_threshold': mask_threshold or 0.3,
+        'pvc_mask_tissues': pvc_mask_tissues or [1, 2, 3],
```

Comment:
We could check that input have the good format, for example that mask_tissues are composed of number between 1 and 7 (or 6, I don't remember)

Reply:
Do you think that we should add an abstract `check_pipeline_parameters` method in Pipeline class?